src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests.Android/Xamarin.Platform.Handlers.DeviceTests.Android.csproj

```diff
@@ -10,8 +10,8 @@
     <TemplateGuid>{122416d6-6b49-4ee2-a1e8-b825f31c79fe}</TemplateGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Xamarin.Platform.Handlers.DeviceTests.Android</RootNamespace>
-    <AssemblyName>Xamarin.Platform.Handlers.DeviceTests.Android</AssemblyName>
+    <RootNamespace>Maui.DeviceTests.Android</RootNamespace>
```

Comment:
If we are chaging this, should we just change the csproj naming also?

Reply:
No we'll do that once we're more set on a name